app/lib/efile/md/md502_calculator.rb

```diff
@@ -420,6 +421,10 @@ def calculate_line_40
           @intake.state_file1099_rs.sum { |item| item.state_tax_withheld_amount.round }
       end
 
+      def calculate_authorize_direct_deposit
+        @intake.bank_authorization_confirmed_yes? ? "X" : nil
+      end
```

Comment:
would love to discuss the following, probably on tuple:

i know that [the original intent of the calculators was for them to mainly hold values that are numbers or booleans used to calculate other lines](https://github.com/codeforamerica/vita-min/pull/3689#issuecomment-1774152046). this doesn't look like it's used downstream anywhere so it could just be determined from the xml builder and left out of the calculator. either way i think i'd prefer not to have a calculator method/line set to the string "X" and instead return a boolean that can be converted to "X" in the xml builder (because "X" is required by an xml type). hopefully that makes sense!

Reply:
hm, update: i went and looked around and i guess this calculator is littered with methods that return "X" lol. i can't think of a reason this functionally matters so idk if i'm going to recommend a refactor? again, we can discuss

Reply:
i asked em the same question when we were pairing on review. i was also curious because there's nothing being calculated here. em had said sometimes these methods are used elsewhere in the calculator for actual calculations. that doesn't seem to be the case here. 

👀 i'm curious what y'all decide post tuple & discussion!

Reply:
upon discussion: let's make this return a boolean & in the XML set `X` or `nil`